.circleci/config.yml

```diff
@@ -72,6 +72,40 @@ jobs:
             - .m2
             - .npm
             - replikativ
+  persistent-set-test:
+    executor: tools/clojurecli
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+      - run:
+          name: Retrieve ssh-fingerprints
+          command: mkdir ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
+          no_output_timeout: 1m
```

Comment:
You only need this when you want to check out a repository with git

Reply:
Ah, I see! Thanks, I have removed that part now for both test jobs.